client/dashboard/app/router.tsx

```diff
@@ -109,6 +110,9 @@ const siteSettingsRoute = createRoute( {
 	path: 'settings',
 	loader: ( { params: { siteSlug } } ) =>
 		queryClient.ensureQueryData( siteSettingsQuery( siteSlug ) ),
+	staticData: {
```

Comment:
I don't have the full picture yet regarding the proper implementation, but here we decouple the route breadcrumb label with the label we use for the menus in the app. Would it be possible and/or preferable to have a single label in the route and extract this in the menus?

Reply:
This is already the case right now, we're duplicating the page title (e.g. `Overview`, `Deployments`) in the menu and in the individual component for that menu. Let me think how to best address this...